package.json.orig

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "aedes",
+  "version": "0.32.0",
+  "description": "Stream-based MQTT broker",
+  "main": "aedes.js",
+  "scripts": {
+    "lint": "standard",
+    "test": "tape test/*.js test/*/*.js | faucet",
+    "ci": "npm run lint; npm run coverage",
+    "coverage": "istanbul cover tape test/*.js test/*/*.js",
+    "coveralls": "cat coverage/lcov.info | coveralls"
+  },
+  "pre-commit": [
+    "lint",
+    "test"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mcollina/aedes.git"
+  },
+  "bugs": {
+    "url": "http://github.com/mcollina/aedes/issues"
+  },
+  "keywords": [
+    "mqtt",
+    "broker",
+    "stream",
+    "internet",
+    "of",
+    "things"
+  ],
+  "author": "Matteo Collina <hello@matteocollina.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "aedes-persistence-redis": "^5.1.0",
+    "compute-mode": "^1.0.0",
+    "concat-stream": "^1.4.7",
+    "convert-hrtime": "^2.0.0",
+    "coveralls": "^3.0.0",
+    "duplexify": "^3.5.1",
+    "faucet": "0.0.1",
+    "istanbul": "^0.4.1",
+<<<<<<< HEAD
+    "mqemitter-redis": "^2.2.0",
+    "mqtt": "^2.13.0",
+    "mqtt-connection": "^3.0.0",
+=======
+    "mqtt": "^2.14.0",
+    "mqtt-connection": "^3.2.0",
+>>>>>>> 9a4eb0449d75a76db91099b32820f18f531a431b
```

Comment:
Should we need this file and conflict code ?

Reply:
no, thanks for noticing